assembl/static2/js/app/index.jsx

```diff
@@ -1,11 +1,27 @@
 import React from 'react';
```

Comment:
Yes, it's better to use jsx extension when we have to use a react component. If not, we have to use js extension. Right?